lib/config.ts

```diff
@@ -6,7 +6,6 @@ export class Configuration extends ConfigBase implements IConfiguration { // Use
 	CI_LOGGER = false;
 	DEBUG = false;
 	TYPESCRIPT_COMPILER_OPTIONS = {};
-	USE_PROXY = false;
```

Comment:
The functionality is introduced as command, so there's no need to manually edit any file. When you use the command, it will create required file (not the config.json ), so there's no need for the user to do anything else - just use the commands.
Where do you think is the appropriate place for such an article?